src/DotNetty.Common/Utilities/TimeUtil.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Common.Utilities
+{
+    using System;
+
+    /// <summary>
+    /// Time utility class.
+    /// </summary>
+    public static class TimeUtil
```

Comment:
I'll see tonight. If I'll change all code, which works with nanoseconds on TimeSpan, I think It will be removed.